core/region.go

```diff
@@ -303,9 +303,7 @@ func (r *region) scanForEnabledAutoScalingGroups() {
 
 	svc := r.services.autoScaling
```

Comment:
You can inline the `r.services.autoScaling` below instead of using this `svc` variable.